R/extract_lhs.R

```diff
@@ -110,32 +147,27 @@ extract_lhs2.glm <- function(model, ital_vars, ...) {
   outcome_escaped <- escape_tex(outcome)
   ss_escaped <- escape_tex(ss)
 
-  if (is.na(ss)) {
-    lhs <- add_tex_ital_v(outcome_escaped, ital_vars)
-  } else {
-    lhs <- paste0(add_tex_ital_v(outcome_escaped, ital_vars),
-                  add_tex_subscripts(
-                    add_tex_ital_v(ss_escaped, ital_vars)
-                  )
-    )
-  }
+  lhs <- add_tex_ital_v(outcome_escaped, ital_vars)
+  p <- paste0("\\operatorname{prob}",
```

Comment:
Is `prob` ideal here? I guess I'm wondering why not just `P`.

Reply:
It can definitely be just `P`. Fixed.